includes/general.admin.inc

```diff
@@ -37,12 +43,22 @@ function islandora_solr_metadata_general_admin_form($form, &$form_state) {
 }
 
 /**
+ * Validation handler for general config form.
+ */
+function islandora_solr_metadata_general_admin_form_validate(&$form, &$form_state) {
```

Comment:
If this validation function exists to make sure there is a separator value entered why not use `#required` on the element itself?